flow/Tracing.h

```diff
@@ -0,0 +1,114 @@
+/*
+ * FBTrace.h
```

Comment:
And TBH I unconsciously expand `FBTrace.h` into `FacebookTrace.h` the moment I saw it LOL

Reply:
It was standing for flatbuffers ;) Initially this was a file that would allow one to write traces in flatbuffers. I removed this for now and will add this functionality in a later PR